ixmp4/core/model.py

```diff
@@ -74,6 +74,9 @@ def list(self, name: str | None = None) -> Iterable[Model]:
     def tabulate(self, name: str | None = None) -> pd.DataFrame:
         return self.backend.models.tabulate(name=name)
 
+    def map(self) -> dict:
+        return self.backend.scenarios.map()
```

Comment:
Is it necessary to expose map to the facade?
Also: typo: should be .models.map(), right?

Reply:
On that note: there should probably be some tests for these.

Reply:
Removed from the facade.